Khan/wonder-blocks
# [breaking change] WB-1029: Add default focus ring a way to override it

## Summary:
The default focus ring is very bare bones.  webapp already has a bunch of focus rings like this in it so I think it's fine to ship this now and then iterator on it later.  This PR also adds a style so that the "pointer" cursor is shown when hovering over "Clickable"s just like regular buttons and links.

Issue: WB-1029

## Test plan:
- yarn start
- load http://localhost:6060/#!/Clickable
- hover over the clickables to see the "pointer" cursor
- tab through the page to see Clickable's default focus rin around the "Client-Side routing" examples

<img width="995" alt="Screen Shot 2021-02-05 at 6 42 01 PM" src="https://user-images.githubusercontent.com/1044413/107100496-89606d80-67e2-11eb-8209-3456183ef3e5.png">
